src/indexset.rs

```diff
@@ -44,6 +44,7 @@ pub type FnvIndexSet<T, N> = IndexSet<T, N, BuildHasherDefault<FnvHasher>>;
 ///     println!("{}", book);
 /// }
 /// ```
+#[derive(Clone)]
```

Comment:
This should be a manual `Clone` implementation. No need for a `Clone` bound on `N`.